coralproject/talk
# [CORL-1495] Improve Cache Busting

## What does this PR do?

When the iframe loads, it adds a time sensitive value to the URL to aid in cache busting. The hash length of assets was also increased.

With these changes I've also modified the embed to remove the requirement of the `Object.assign` polyfill, which saved us some precious KB on the `embed.js` file.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?

1. Run `npm run build` and see the increased length of built assets.
2. Load a page with the iframe embed on it and see a time sensitive value on the URL for it.
3. Load the page on IE 11 and ensure it still works.
